static/src/components/grabNumber.jsx

```diff
@@ -0,0 +1,12 @@
+'use strict';
+import React from 'react';
+import {render} from 'react-dom';
+
+class GrabNumber extends React.Component{
```

Comment:
I didn't use it, since it seemed easier just to have everything in the fetch class. I wasn't sure if you still wanted to have the number array in a different file or not... but I can change it however needed.